pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/UpsertTableSegmentUploadIntegrationTest.java

```diff
@@ -74,7 +74,7 @@ public void setUp()
     // Create and upload schema and table config
     Schema schema = createSchema();
     addSchema(schema);
-    TableConfig tableConfig = createUpsertTableConfig(avroFiles.get(0), PRIMARY_KEY_COL, getNumKafkaPartitions());
+    TableConfig tableConfig = createUpsertTableConfig(avroFiles.get(0), PRIMARY_KEY_COL, null, getNumKafkaPartitions());
```

Comment:
Consider merging this into the `UpsertTableIntegrationTest`? Essentially we want to upload one segment and check if it works properly

Reply:
The dataset used in that class is not really "upsert" like data. It has the airline data set with some 100K rows. I just wanted a simple functional test that is easy to read and doesn't require a large dataset. I can move the tests into a single class, if that's what you want.